ansible/roles/test/tasks/acltb_scale.yml

```diff
@@ -0,0 +1,79 @@
+# Set facts for the loganalizer
+- set_fact:
+    out_dir: /tmp/
+    tests_location: roles/test/tasks
+    testname: acl_scale
+    run_dir: /tmp/
+
+# Copy ACL config to the switch
+- name: Copy ACL config file to the DUT
+  copy: src="roles/test/tasks/acl/{{ item }}" dest="/tmp/"
+  with_items:
+    - "acltb_test_rules-del.json"
+
+- block:
+    - name: change CRM polling interval
+      command: crm config polling interval 10 
+
+    - pause:
+        seconds: 10 
+
+    - name: Get the ACL usage via CRM
+      command: crm show resources acl table
+      register: out
+
+    - name: Extract out the data acl info from CRM output
+      set_fact:
+        data_acl_crm_info: "{{out.stdout_lines[4].split()}}"
+
+    # scale_number is the number of rules the template file will generate
+    - name: Extract out the total available acl entry count from CRM show
+      set_fact:
+        scale_number: "{{data_acl_crm_info[3] | int}}"
+
+    - debug:
+        msg: "scale number is {{scale_number}}" 
+    
+    - name: Copy acl rules file
+      become: true
+      template: src=acl_scale_rules.j2 
+                dest=/tmp/acltb_scale_rules.json
+    
+    - name: Apply scalable rule set
+      vars:
+        command_to_run: "acl-loader update full /tmp/acltb_scale_rules.json"
+        test_expect_file: expect_acl_scale_failure
+        errors_expected: true
+      include: roles/test/tasks/run_command_with_log_analyzer.yml
+
+    - pause:
+        seconds: 90 
+
+    - name: Get the ACL usage via CRM
+      command: crm show resources acl table
+      register: out
+
+    - name: Extract out the data acl info from CRM output
+      set_fact:
+        data_acl_crm_info: "{{out.stdout_lines[4].split()}}"
+
+    - name: Extract out the total available acl entry count from CRM show
+      set_fact:
+        available_count: "{{data_acl_crm_info[2] | int}}"
+
+    - name: Print output
+      debug:
+        msg: "This DUT can scale up to {{ used_count }} ACL entries"
+      when: available_count = 0
+
+    - name: Print error output
+      debug:
+        msg: "CRM module still shows some available entries while SAI_INSUFFICIENT_ERROR shown"
+      when: available_count != 0
+
+- always:
+    - name: Clean up ACL rules.
+      vars:
+        command_to_run: "acl-loader update full /tmp/acltb_test_rules-del.json"
+        errors_expected: false
+      include: roles/test/tasks/run_command_with_log_analyzer.yml
```

Comment:
Could you consider checking remaining ACL entry count from CRM again after the ACL rules are deleted?